### DISCUSSION TITLE
Model saving in Avalanche

### DISCUSSION THREAD

**Discussion by AdityaJainTD** [ContinualAI/avalanche#1068] [Q&A] [answered]
Hi,

I was wondering, how do we save the model in Avalanche? Is it same as saving torch model? An example would be great.

**Comment by AndreaCossu** [comment, marked as answer]
Yes, you can save/load the model as you would normally do in PyTorch (`torch.save()`). If you want to do that within the strategy you can sub-class it and override a specific callback like `after_training_iteration`. Otherwise you can save the model in your main, like:
```python
for exp in benchmark.train_stream:
    strategy.train(exp)
    strategy.eval(benchmark.test_stream)
    # here you can save your model. You can also access it by strategy.model property
    # in this way, you have a checkpoint after each training experience
```
